src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <!-- Keep these in sync with _ILLinkTasksTFM in Sdk.props. -->
```

Comment:
Sdk.props also needs to be updated so that MSBuild picks up the right version of the task.

Reply:
Done